src/uu/ls/src/ls.rs

```diff
@@ -297,8 +297,11 @@ impl TimeStyler {
             // So it's not yet implemented
             TimeStyle::Locale => StrftimeItems::new("%b %e  %Y").parse(),
             TimeStyle::Format(fmt) => {
-                // TODO (#7802): Replace with new_lenient
-                StrftimeItems::new(custom_tz_fmt::custom_time_format(fmt).as_str()).parse_to_owned()
+                // Box the fmt value to make it static.
+                StrftimeItems::new_lenient(Box::leak(
```

Comment:
Why the `Box::leak` change?

Reply:
Oh, my bad. Was too concentrated on `parse()`, missed that it can be done much simpler